examples/predict.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+###################################################
+# Assume the user already ran download_and_train.py
+###################################################
+
+gpu_id = 0
+
+image_save_dir = '{}/images/'.format(os.getcwd())
+model_save_dir = '{}/model/'.format(os.getcwd())
+prefs_save_path = '{}/prefs.json'.format(os.getcwd())
+
+data_save_path_test = '{}/image_list_test.csv'.format(image_save_dir)
+
+command_str = 'python  ../fnet/cli/predict.py ./ ' \
```

Comment:
Similar for the train script, the user should probably call `fnet predict` instead. Also, I still would like to see some justification for creating a Python script that creates a bash script to be run in Python. Why not have this as a bash script?